vertical-pod-autoscaler/recommender/recommender.go

```diff
@@ -0,0 +1,71 @@
+package main
+
+import (
+	"fmt"
+	"time"
+
+	"k8s.io/apimachinery/pkg/fields"
+	"k8s.io/autoscaler/vertical-pod-autoscaler/recommender/metrics"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/cache"
+	"k8s.io/contrib/service-loadbalancer/Godeps/_workspace/src/github.com/golang/glog"
+	apiv1 "k8s.io/kubernetes/pkg/api/v1"
+	kube_client "k8s.io/kubernetes/pkg/client/clientset_generated/clientset"
+	v1lister "k8s.io/kubernetes/pkg/client/listers/core/v1"
+	resourceclient "k8s.io/metrics/pkg/client/clientset_generated/clientset/typed/metrics/v1alpha1"
+)
+
+// Recommender recommend resources for certain containers, based on utilization periodically got from metrics api.
+type Recommender interface {
+	Run()
+}
+
+type recommender struct {
+	metricsClient           metrics.Client
+	metricsFetchingInterval time.Duration
+}
+
+func (r *recommender) Run() {
+	utilizations, err := r.metricsClient.GetContainersUtilization()
+	if err != nil {
+		glog.Errorf("Cannot get containers utilization. Reason: %+v", err)
+	}
+	for n, utilization := range utilizations {
+		fmt.Printf("Utilization #%v: %+v", n, utilization)
```

Comment:
This method is only temporary and it is here to generate any kind of output from the client. When actual recommender functionality is created, I expect it will be removed.

But yes, I can replace it with logging :)